client/src/components/header/Header.js

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect } from 'react';
+import Box from '@mui/material/Box';
+import styled from '@emotion/styled';
+import { DatePicker, LocalizationProvider } from '@mui/x-date-pickers';
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { Store } from '../../utils/store';
+import { Typography } from '@mui/material';
+import Budget from '../../APIs/budget';
+import AddTransaction from './TransactoinModel';
+
+function Navbar() {
+  const store = Store.useContainer();
+
+  const month = store.month.getMonth();
+  const year = store.month.getFullYear();
+
+  const monthName = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ];
+
+  useEffect(() => {
+    const getBudget = async () => {
+      try {
+        const data = await Budget.get(`${month}${year}`);
+        store.setTransactions(data.budget?.Transactions || []);
+        store.setBudget(data.budget);
+      } catch (error) {}
+    };
+
+    getBudget();
```

Comment:
why did you call the same function (getBudget) twice inside the useEffect?

Reply:
Fixed issues in useEffects in commit d97d6a5f54186a491fb8e2f6b579d8f813ed86e3